docs/storing-querying/querying-data/index.md

```diff
@@ -186,52 +201,80 @@ Each type of entity is in a dedicated `ARRAY`-type column. The column name will
 You can query a single entity’s fields by extracting them like so:
 
 ```sql
-select
+SELECT
     ...
     contexts_my_entity_1[0]:myField::varchar,  -- field will be variant type so important to cast
     ...
-from 
+FROM
     atomic.events
 ```
 
 Alternatively, you can use the [`lateral flatten`](https://docs.snowflake.com/en/sql-reference/functions/flatten) function to explode out the array into one row per entity value.
 
 ```sql
-select
+SELECT
     ...
     r.value:myField::varchar,  -- field will be variant type so important to cast
     ...
-from 
-    atomic.events  as t,
+FROM
+    atomic.events AS t,
     LATERAL FLATTEN(input => t.contexts_my_entity_1) r
 ```
 
 </TabItem>
-<TabItem value="databricks" label="Databricks">
+<TabItem value="databricks" label="Databricks, Spark SQL">
 
 Each type of entity is in a dedicated `ARRAY<STRUCT>`-type column. The column name and the fields in the `STRUCT` will be determined by the entity’s schema. See [how schemas translate to the warehouse](/docs/storing-querying/schemas-in-warehouse/index.md) for more details.
 
 You can query a single entity’s fields by extracting them like so:
 
 ```sql
-select
+SELECT
     ...
     contexts_my_entity_1[0].my_field,
     ...
-from 
+FROM
     atomic.events
 ```
 
 Alternatively, you can use the [`LATERAL VIEW`](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-qry-select-lateral-view.html) clause combined with [`EXPLODE`](https://docs.databricks.com/sql/language-manual/functions/explode.html) to explode out the array into one row per entity value.
 
 ```sql
-select
+SELECT
     ...
     my_ent.my_field,
     ...
-from 
+FROM
     atomic.events
-    lateral view explode(contexts_my_entity_1) as my_ent
+    LATERAL VIEW EXPLODE(contexts_my_entity_1) AS my_ent
+```
+
+</TabItem>
+<TabItem value="synapse" label="Synapse Analytics 🧪">
+
+Each type of entity is in a dedicated column in JSON format. The column name will be determined by the entity’s schema. See [how schemas translate to the warehouse](/docs/storing-querying/schemas-in-warehouse/index.md) for more details.
+
+You can query a single entity’s fields by extracting them like so:
+
+```sql
+SELECT
+    ...
+    JSON_VALUE(contexts_my_entity_1, '$[0].myField')
+    ...
+FROM
+    OPENROWSET(...) AS [result]
+```
```

Comment:
I can't actually test this and haven't used T-SQL before, so I assume it's working.

Reply:
I have tested it